students/Harry_Maher/Session08/circle_class.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
```

Comment:
You might also want to check out this page for an easier way to define all the formulas needed for a circle: 

    https://docs.python.org/3.6/library/functools.html#functools.total_ordering